src/kili/adapters/kili_api_gateway/organization/types.py

```diff
@@ -0,0 +1,23 @@
+"""Organization types for Kili API Gateway."""
+
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass(frozen=True)
+class KiliAPIGateWayCreateOrganizationInput:
+    """Organization to create Kili API Gateway input."""
+
+    name: str
+    address: str
+    city: str
+    country: str
+    zip_code: str
+
+
+@dataclass(frozen=True)
+class KiliAPIGateWayUpdateOrganizationInput:
+    """Organization to update Kili API Gateway input."""
+
+    license: Optional[dict] = None  # noqa: A003
+    name: Optional[str] = None
```

Comment:
input OrganizationData {
  license: String
  name: String
  matchingDomain: String
}

Reply:
license is a string

Reply:
oh my bad, it's ok if there is a json.dumps after in the mapper